packages/cf-style-provider/src/createRenderer.js

```diff
@@ -47,6 +48,7 @@ const createRenderer = opts => {
     enhancers,
     selectorPrefix: usedOpts.selectorPrefix,
     mediaQueryOrder: [
```

Comment:
This list seems weird and redundant. Can't this just be `Object.values(mediaQueries).map(removePrefix)`?